### DISCUSSION TITLE
Cabrillo file...

### DISCUSSION THREAD

**Discussion by OH9FKJ** [mbridak/not1mm#64] [General]
My cabrillo file does not print any QSOs at all, only the information section. Has anyone else noticed the same thing?
The Adif file prints correctly and you can then convert it online, but it would be easier without that...
 -  73 de OH9FKJ

**Comment by mbridak** [comment]
Hi, what contest was the cabrillo for?

Could you start not1mm from a terminal and see if it spits out an error while you're generating the cabrillo?

**Comment by awandahl** [comment]
Most likely a funny character like å, ä or ö in the station settings. This was the reason in my case.
